src/client/app/frameworks/core.framework/services/localforage.service.ts

```diff
@@ -0,0 +1,12 @@
+import {Injectable} from '@angular/core';
+
+const localForage:LocalForage = require('localforage');
+
+@Injectable()
+export class LocalForageService {
```

Comment:
I would rename this service something more generic though so as not to tie it to the underlying implementation. Makes it easier for others to swap out under the hood with something else.

So maybe `PersistenceService`, `DBService`, `CacheService`, `LocalCacheService`, `LocalStorageService`, etc...  Happy to hear other thoughts...